index.html

```diff
@@ -28,89 +28,206 @@
+        It would act as a hint of the intended priority rather than an instruction to the browser.</li>
+    </ul>
+
+    <p>With this attribute, the browser should make an effort to respect the developer's preference for the importance of a
+      resource when fetching it. Note that this is intentionally weak language, allowing for a browser to apply its own preferences
+      for resource priority or heuristics if deemed important.
+    </p>
+    <p>
+      <div class="note">
+        <p>Priority Hints compliment existing browser loading primitives such as preload. Preload is a mandatory and high-priority
+          fetch for a resource that is necessary for the current navigation. Priority Hints can hint that a resource's priority should be lower or higher than its default, and can also be used to provide more granular prioritization to preloads.</p>
+      </div>
+    </p>
+    <p>This is how we conceptually think about different resource types under the hood in browsers today. It may translate well
+      to user-space where different types of content share similar properties.</p>
+
+    <h2 id="examples">Examples</h2>
+
+    <strong>Lower fetch priority when browser defaults are insufficient</strong>
+    <p>Browsers make a best-effort to assign a reasonable fetch priority to images so those in-viewport appear as soon as possible.
+      In many cases, this will not cause issues, however, what if some above-the-fold imagery is not as critical as others?
+      Priority Hints can assist here. </p>
+
+    <div class="example" title="Example 1">
+      <p>FastCorp Inc. have a carousel of images at the top of a page with the first image visible and the remaining images
+        images off screen. The markup of this carousel may look as follows:</p>
+      <pre highlight="html">
+&lt;ul class=&quot;carousel&quot;&gt;
+<br/> &lt;!-- The first image is visible --&gt;
+<br/> &lt;img src=&quot;img/carousel-1.jpg&quot;&gt;
+<br/> &lt;!-- The other carousel images are not --&gt;
+<br/> &lt;img src=&quot;img/carousel-2.jpg&quot;&gt;
+<br/> &lt;img src=&quot;img/carousel-3.jpg&quot;&gt;
+<br/> &lt;img src=&quot;img/carousel-4.jpg&quot;&gt;
+<br/>&lt;/ul&gt;
+</div>
 
-&lt;script src=&quot;non-critical-1.js&quot; async importance=&quot;low&quot;&gt;&lt;/script&gt;
+<p>Browser heuristics may cause all four images to be fetched with a high priprity, despite three of them not initially being visible. This may cause these fetches to delay the first image from completing fetching or contend with fetches for other in-viewport resources.</p>
+<p>A developer could use rel=preload to preload the first image in the carousel, however doing so may not provide the outcome one expects: rel=preload will effectively prioritize that image above everything else, and if that image is large, it may block rendering as even stylesheets may end up waiting for that image to complete downloading. Priority Hints may help here.</p>
```

Comment:
Yeah, the fact that it can be started earlier than other potentially more important things seems to be the thing to drive at. In a perfect world it seems that we'd want to be able to use preload for early discovery, but have the browser be smart enough (via perhaps some sort of queue) to make sure that we can get the earliest discovery possible _without_ getting in the way of higher prio stuff.